projects/plugins/jetpack/_inc/client/components/apps-card/index.jsx

```diff
@@ -24,17 +23,40 @@ class AppsCard extends React.Component {
 		} );
 	};
 
-	dismissCard = () => {
-		this.props.dismissAppCard();
-		analytics.tracks.recordJetpackClick( {
-			target: 'apps-card',
-			button: 'dismiss',
-			page: this.props.location.pathname,
-		} );
-	};
+	getAppCards = () => (
+		<div className="jp-apps-card__apps-badges">
+			<AppsBadge
+				altText={ __( 'Google Play Store download badge.', 'jetpack' ) }
+				titleText={ __( 'Download the Jetpack Android mobile app.', 'jetpack' ) }
+				storeName="android"
+				storeLink="https://play.google.com/store/apps/details?id=com.jetpack.android&utm_source=jpdash&utm_medium=cta&utm_campaign=getappscard"
+				onBadgeClick={ this.trackDownloadClick }
+			/>
+			<AppsBadge
+				altText={ __( 'Apple App Store download badge.', 'jetpack' ) }
+				titleText={ __( 'Download the Jetpack iOS mobile app.', 'jetpack' ) }
+				storeName="ios"
+				storeLink="https://apps.apple.com/us/app/jetpack-wp-security-speed/id1565481562?pt=299112ct=jpdash&mt=8"
+				onBadgeClick={ this.trackDownloadClick }
+			/>
+		</div>
+	);
+
+	getQrCode = () => (
+		<div className="jp-apps-card__apps-qr-code">
+			<img src={ imagePath + 'get-apps-qr-code.svg' } alt="" />
+			<p className="jp-apps-card__caption">
+				{ __( 'Visit', 'jetpack' ) }{ ' ' }
+				<a className="jp-apps-card__link" href="https://jetpack.com/app">
+					jetpack.com/app
+				</a>
+				{ __( ' or scan this code to download the Jetpack mobile app.', 'jetpack' ) }
```

Comment:
Explicitly directing users to use their mobile device may help avoid confusion that the URL does not work for non-mobile devices. pcdRpT-SK-p2#comment-1112 
```suggestion
				{ __( ' from your mobile device or scan this code to download the Jetpack mobile app.', 'jetpack' ) }
```